libs/designsystem/src/lib/components/dropdown/dropdown.component.spec.ts

```diff
@@ -253,6 +258,33 @@ describe('DropdownComponent', () => {
       });
     });
 
+    fdescribe('when configured with cardAligment', () => {
```

Comment:
`fdescribe` --> `describe` without the `f` prefix

Reply:
Doh.. removed :)